src/index.ts

```diff
@@ -68,6 +69,29 @@ export class ServerlessConventions {
           return errors;
      }
 
+     // Cloud formation service role validation
+     // Confirm that the service contains an iam deployment role
+     checkIAMDeploymentRole(service: Service) : Array<string> {
+          let errors : Array<string> = [];
+          const regex = new RegExp('^arn:aws:iam::\\d{12}:role\\/.+');
+
+          // This is a bit of a messy way to get the iam
+          // Ideally it would be typed in @types/serverless
+          const iam = (<any>service.provider).iam;
+          
+          if (iam === undefined || iam === null) {
```

Comment:
You should be able to replace this with (iam == null) abstract equality will evaluate `null` and `undefined` the same.

Reply:
Updated :+1: